pkg/ebpf/event_common.go

```diff
@@ -127,6 +127,11 @@ func (c ConnectionStats) String() string {
 }
 
 // ByteKey returns a unique key for this connection represented as a byte array
+// It's as following:
```

Comment:
Nice, thanks for adding this

Reply:
While you're here, thoughts on merging family + type into a single `4b`?

Reply:
We can but we will still have to write a whole `byte` for those 4bits